tests/test_typeform_all_fields.py

```diff
@@ -66,6 +75,7 @@ def test_run(self):
                 self.assertTrue(expected_automatic_keys.issubset(
                     expected_all_keys), msg='{} is not in "expected_all_keys"'.format(expected_automatic_keys-expected_all_keys))
```

Comment:
This assertion is duplicated on L#88, remove one of the assertion.

Reply:
Removed duplicate assertion.